### PR TITLE
Ltd 4047 approve refuse picklist update

### PR DESCRIPTION
**Pull request by Tllew**
Convert picklists to radio + textareas in the approval/refusals workflow

[LTD-4047](https://uktrade.atlassian.net/browse/LTD-4047)

[LTD-4047]: https://uktrade.atlassian.net/browse/LTD-4047?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: caseworker/advice/views.py [hunk 1]
```diff
@@ -176,8 +175,16 @@ def get_form_kwargs(self):
             self.request, type="standard_advice", disable_pagination=True, show_deactivated=False
         )
         proviso = get_picklists_list(self.request, type="proviso", disable_pagination=True, show_deactivated=False)
+
+        footnote_details = get_picklists_list(
+            self.request, type="footnotes", disable_pagination=True, show_deactivated=False
+        )
+
+        #     picklist_attrs={"target": "footnote_details", "type": "footnotes", "name": "reporting footnote"},
```
Comment (currycoder) [inline caseworker/advice/views.py:183]: Can we remove this commented code?

File: caseworker/advice/views.py [hunk 2]
```diff
@@ -212,6 +219,13 @@ def get_form(self):
         else:
             return forms.RefusalAdviceForm(choices, **self.get_form_kwargs())
 
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs["refusal_reasons"] = get_picklists_list(
+            self.request, type="standard_advice", disable_pagination=True, show_deactivated=False
```
Comment (currycoder) [inline caseworker/advice/views.py:225]: I find it interesting that all of this is `type="standard_advice"`..  I realise that it's probably beyond scope, but it feels like there should be different picklist options for approval/refusal.

File: caseworker/advice/views.py [hunk 3]
```diff
@@ -289,13 +303,21 @@ def get_form(self):
...
+            return forms.get_approval_advice_form_factory(
+                advice, approval_reason, proviso, footnote_details, self.request.POST
+            )
         elif advice["type"]["key"] == "refuse":
             self.template_name = "advice/refusal_advice.html"
             denial_reasons = get_denial_reasons(self.request)
             choices = group_denial_reasons(denial_reasons)
-
-            return forms.get_refusal_advice_form_factory(advice, choices, self.request.POST)
+            refusal_reasons = get_picklists_list(
```
Comment (currycoder) [inline caseworker/advice/views.py:317]: Can this be factored out above the if statement to keep things more DRY?

**Review by currycoder — COMMENTED** [review]
Some minor comments inline.

File: caseworker/advice/views.py (hunk 3 above)
Comment (depsiatwal) [reply caseworker/advice/views.py:317]: agree with this other then that LGTM

File: caseworker/advice/views.py (hunk 2 above)
Comment (Tllew) [reply caseworker/advice/views.py:225]: agreed, I do think it's a hack due to the team the user is on.

**Review by depsiatwal — APPROVED** [review]
lgtm

**Pull request merged**